client/src/project/new/ProjectNew.present.js

```diff
@@ -211,1074 +191,114 @@ function ForkProjectContent(props) {
     <Fragment>
       <Title handlers={handlers} input={input} meta={meta} />
       <Namespaces handlers={handlers} input={input} namespaces={namespaces} user={user} />
-      <Home input={input} />
+      <ProjectIdentifier input={input} />
       <Visibility handlers={handlers} input={input} meta={meta}/>
     </Fragment>
   );
 }
 
+const NewProjectForm = (
+  { automated, config, handlers, input, meta, namespaces, namespace,
+    user, importingDataset, userRepo, templates } ) => {
+
+  const isFormProcessingOrFinished = (meta) => {
+    // posting
+    if (meta.creation.creating || meta.creation.projectUpdating || meta.creation.kgUpdating)
+      return true;
+    // posted successfully with visibility or KG warning
+    return !!(meta.creation.created && (meta.creation.projectError || meta.creation.kgError));
```

Comment:
This is clever, but I would prefer code that is a bit "dumber." :)

```suggestion
    if (meta.creation.created) return true;
    return (meta.creation.projectError || meta.creation.kgError);
```